src/usr/local/www/interfaces_bridge_edit.php

```diff
@@ -184,6 +184,20 @@ function is_aoadv_used($pconfig) {
 		$input_errors[] = gettext("At least one member interface must be selected for a bridge.");
 	}
 
+	if (is_array($_POST['members']) && is_array($config['captiveportal'])) {
+		foreach ($_POST['members'] as $member) {
+			foreach ($config['captiveportal'] as $cp) {
+				foreach (explode(',', $cp['interface']) as $cpint) {
+					if ($member == $cpint) {
```

Comment:
This last foreach + if could be replaced by a single statement like this:

`if (in_array($member, explode(',', $cp['interface']))`

Reply:
Thanks, replaced